new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleNginxValidation.java

```diff
@@ -293,76 +304,110 @@ public void testCreateMiiDomain() {
-      logger.info("Check managed server service {0} is created in namespace {1}",
-          managedServerPrefix + i, domainNamespace);
-      checkServiceCreated(managedServerPrefix + i);
+    // check the readiness for the managed servers in each cluster
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS; i++) {
+      for (int j = 1; j <= replicaCount; j++) {
+        String managedServerPodName =
+            domainUid + "-" + CLUSTER_NAME_PREFIX + i + managedServerNameBase + j;
+
+        // check managed server pod is created
+        logger.info("Check for managed server pod {0} is created in namespace {1}",
+            managedServerPodName, domainNamespace);
+        checkPodCreated(managedServerPodName);
+
+        // check managed server pod is ready
+        logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
+            managedServerPodName, domainNamespace);
+        checkPodReady(managedServerPodName);
+
+        // check managed server service is created
+        logger.info("Check managed server service {0} is created in namespace {1}",
+            managedServerPodName, domainNamespace);
+        checkServiceCreated(managedServerPodName);
+      }
     }
   }
 
   @Test
   @Order(2)
-  @DisplayName("Create an ingress for a WebLogic domain in the specified domain namespace")
+  @DisplayName("Create an ingress for each cluster of the WebLogic domain in the specified domain namespace")
   public void testCreateIngress() {
 
-    // create an ingress in domain namespace
-    assertThat(assertDoesNotThrow(() -> createIngress(domainNamespace, domainUid, clusterName, managedServerPort)))
-            .as("createIngress succeeds")
-            .withFailMessage(String.format("failed to create an ingress for domain %s in namespace %s",
-                domainUid, domainNamespace))
-            .isTrue();
-
-    // check the ingress is created
-    String ingressName = domainUid + "-nginx";
-    assertThat(assertDoesNotThrow(() -> getIngressList(domainNamespace)))
-        .as(String.format("found the ingress %s in namespace %s", ingressName, domainNamespace))
-        .withFailMessage(String.format("can not find ingress %s in namespace %s", ingressName, domainNamespace))
-        .contains(ingressName);
-
-    logger.info("ingress is created in namespace {0}", domainNamespace);
+    // create an ingress for each cluster of the domain in the domain namespace
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS; i++) {
+
+      String clusterName = CLUSTER_NAME_PREFIX + i;
+      String ingressName = domainUid + "-" + clusterName + "-nginx";
+
+      logger.info("Creating ingress {0} for cluster {1} of domain {2} in namespace {3}",
+          ingressName, clusterName, domainUid, domainNamespace);
+      assertThat(createIngress(ingressName, domainNamespace, domainUid, clusterName,
+          MANAGED_SERVER_PORT, domainUid + "." + clusterName + ".test"))
+          .as("Test createIngress succeeds")
```

Comment:
Descriptable.as() java doc is here: https://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/Descriptable.html#as-java.lang.String-java.lang.Object...-

The msg in as() sets the description of the assertion that is going to be called after.
It is not the message printed if the assertion passes.

If the assertion fails, in your case, we will get the message something like:
[Creating Ingress <ingress name> for cluster <cluster name> succeeded] {failed to create an ingress for cluster <cluster-name> of domain <domain_name> in namespace }

It will be confusing. 
How about change to:

Checking that ingress {0} for cluster {1} was created successfully.